.github/workflows/check-test-release.yml

```diff
@@ -114,6 +114,17 @@ jobs:
         GITHUB_MATRIX_PYTHON: ${{ matrix.python }}
         BITBUCKET_USERNAME: ${{ secrets.BITBUCKET_USERNAME }}
         BITBUCKET_PASSWORD: ${{ secrets.BITBUCKET_PASSWORD }}
+    - name: Run Heroku tests
+      if: contains(github.event.pull_request.title, 'heroku')
+      timeout-minutes: 40
+      run: pytest tests/contrib/test_heroku.py
+      env:
+        GITHUB_USERNAME: ${{ secrets.GH_USERNAME }}
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
```

Comment:
Ah, you didn't delete `HEROKU_API_KEY` and `HEROKU_TEAM` for the step running `pytest -k 'not heroku'`. I guess you should.

Reply:
done, but don't merge it yet, will retest those 2 dummy PRs I opened